Header header
float32[] direction
float32[] dist
uint32[] type
float32[] orientation
uint32[] uid
